_data/sac_2025.yaml

```diff
@@ -0,0 +1,88 @@
+- affiliation: "Université de Montréal"
+  bio: "I am a PhD student working with Vincent Taschereau-Dumouchel on bridging the gap between quantitative and qualitative approaches in cognitive neuroscience and computational psychiatry. During my master's, I worked on mapping out subjective experience using a combination of psychometrics, NLP and generative AI, with a focus on accelerating mental health care research. I now aim to develop innovative applications of AI to the study of brains and consciousness. Being part of the UNIQUE SAC, as well as the broader neuroAI community, has been a huge source of inspiration and motivation, which is why I'm eager to continue on this path for a second year."
+  committee: Co-Chair of the SAC
+  email: charlotte.volk@mail.mcgill.ca
+  fullname: shawn.manuel@umontreal.ca
+  photo: shawn_manuel.jpg
+  alttext: none
+  pronouns: he/him
+  twitter: https://x.com/shwnmnl
+  website: https://shwnmnl.github.io/resume.html
+
+- affiliation: "McGill University and Mila"
+  bio: "I am a Master's student in Computer Science at McGill and Mila, supervised by Blake Richards and Shahab Bakhtiari. I'm interested in leveraging AI models to study human learning processes. In my free time, I love to read, figure skating, and crochet. "
+  committee: Co-Chair of the SAC
+  email: charlotte.volk@mail.mcgill.ca
+  fullname: Charlotte Volk
+  photo: charlotte_volk.png
+  alttext: none
+  pronouns: she/her
+
+- affiliation: "Université de Montréal and Mila"
+  bio: "I am a PhD student in Psychology at Université de Montréal under the guidance of Prof. Shahab Bakhtiari. My research is at intersection of artificial intelligence and neuroscience, particularly focusing on modelling the visual system. I use artificial neural networks trained with self-supervised learning to simulate and understand the complexities of the visual system."
+  committee: Technical Lead
+  email: cornelius.crijnen@umontreal.ca
+  fullname: Cornelius Arjan Jacques Crijnen
+  photo: cornelius_crijnen.jpeg
+  alttext: none
+  pronouns: he/him
+
+- affiliation: "Université de Montréal and Mila"
+  bio: "I am a master's student at Mila and Université de Montréal with Matthew Perich and Guillaume Lajoie. I am intrigued by how AI systems can rapidly learn and generalize to new tasks given little to no information. To this end, I aim to mathematically elucidate how different paradigms of task learning (e.g., continual, meta, in-context, etc.) are affected by factors such as learning dynamics, compositional representations, and memory."
+  committee: Liaison to the Equity, Diversity, and Inclusion Committee
+  email: hee-woon.ryoo@umontreal.ca
+  fullname: Avery Ryoo
+  photo: avery_ryoo.jpeg
```

Comment:
If there's an existing picture of Avery, could you just change the filename to be the same as the old one? Or modify the 2024 team file to use just the same picture (and delete any duplicates).